src/geometry/Shape.ts

```diff
@@ -53,11 +53,29 @@ export default abstract class Shape extends THREE.Group {
       stroke?: boolean;
       fill?: boolean;
       closed?: boolean;
+      position?: THREE.Vector3;
+      rotation?: number;
+      scale?: THREE.Vector3;
       fillPoints?: Array<THREE.Vector3>;
     } = {},
   ) {
     super();
     config = Object.assign(Shape.defaultStyle(), config);
+    
+    // Apply position, rotation, and scale if provided
+    if (config.position) {
+      this.position.copy(config.position);
+    }
+    if (config.rotation !== undefined) {
```

Comment:
Have the rotation parameter take a`THREE.Euler | number` and if it's a number treat it as rotation around the z-axis.